Gemfile

```diff
@@ -61,8 +62,7 @@ gem 'prawn'
 gem 'pundit'
 gem 'rack-attack'
 gem 'rack-cors', require: 'rack/cors'
-gem 'rails-api'
```

Comment:
This gem was merged into Rails itself, so I believe the migration path is to `config.api_only = true` in our `application.rb`.